OneCommunityGlobal/HGNRest
# Luis add new warnings for no summary and not enough hours backend

# Description
This is the backend for adding two new warnings to the warning tracker. 



## Related PRS (if any):
Follow the instructions on the frontend
To test this backend PR you need to checkout the [#2954](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2954)


## Main changes explained:
--Updated the warning's enum to accept the new warnings 
--Created new route to get the special warnings as I designed them to have a new flag. 
-- Also integrated the email functionality to send an email after receiving 3 warnings

## How to test:
1. check into current branch
2. do `npm run build` and `npm run start` to run this PR locally
3. Clear site data/cache


## Note:
Include the information the reviewers need to know.
